### ISSUE TITLE
Initial migration for django 1.7+ is incomplete.

### ISSUE THREAD

**Issue by Mihara** [gizmag/django-fsm-log#17]
I've been running around upgrading from django 1.6 and this came up:

    kursitet@precise64:~/kursitet-platform$ ./manage.py makemigrations django_fsm_log -v3 --dry-run
    Migrations for 'django_fsm_log':
      0002_auto_20150620_1353.py:
        - Change Meta options on statelog
    Full migrations file '0002_auto_20150620_1353.py':
    # -*- coding: utf-8 -*-
    from __future__ import unicode_literals
    
    from django.db import models, migrations
    
    
    class Migration(migrations.Migration):
    
        dependencies = [
            ('django_fsm_log', '0001_initial'),
        ]
    
        operations = [
            migrations.AlterModelOptions(
                name='statelog',
                options={'get_latest_by': 'timestamp'},
            ),
        ]

Basically, it appears that the meta options are not reflected in the migration when Django wants them to be. This is only a thing in Django 1.8, I believe, as when upgrading to 1.7 the warning about model changes not being reflected in migrations did not come up.

**Comment by dmugtasimov** [comment]
please, fix

**Comment by hzy** [comment]
I would be happy to accept a pull request to add the required migration
